x/auth/keeper/keeper.go

```diff
@@ -92,7 +92,7 @@ func NewAccountKeeper(
 
 // Logger returns a module-specific logger.
 func (ak AccountKeeper) Logger(ctx sdk.Context) log.Logger {
-	return ctx.Logger().With("module", "x/"+types.ModuleName)
+	return ctx.Logger().Level(ctx.GetLogLevel(types.ModuleName)).With("module", "x/"+types.ModuleName)
```

Comment:
could you not filter based off of module=staking, this is how its done with tendermint and sdk right now (main)